autoload/ale/fixers/erlfmt.vim

```diff
@@ -4,18 +4,49 @@
 call ale#Set('erlang_erlfmt_executable', 'erlfmt')
 call ale#Set('erlang_erlfmt_use_global', get(g:, 'ale_use_global_executables', 0))
 call ale#Set('erlang_erlfmt_options', '')
+call ale#Set('erlang_erlfmt_use_rebar', 0)
+
+call ale#Set('erlang_rebar_executable', 'rebar3')
+call ale#Set('erlang_rebar_use_global', get(g:, 'ale_use_global_executables', 0))
 
 function! ale#fixers#erlfmt#GetExecutable(buffer) abort
     return ale#path#FindExecutable(a:buffer, 'erlang_erlfmt', ['erlfmt'])
 endfunction
 
+function! ale#fixers#erlfmt#GetRebarExecutable(buffer) abort
+    return ale#path#FindExecutable(a:buffer, 'erlang_rebar', ['rebar3'])
+endfunction
+
 function! ale#fixers#erlfmt#Fix(buffer) abort
     let l:options = ale#Var(a:buffer, 'erlang_erlfmt_options')
-    let l:executable = ale#fixers#erlfmt#GetExecutable(a:buffer)
+    let l:use_rebar = ale#Var(a:buffer, 'erlang_erlfmt_use_rebar')
+
+    if l:use_rebar
```

Comment:
It would be clearer if the two cases were handled in separate functions, like this:

    function! ale#fixers#erlfmt#Fix(buffer) abort
        return ale#Var(a:buffer, 'erlang_erlfmt_use_rebar3')
        \   ? s:FixWithRebar3(a:buffer)
        \   : s:FixWithEscript(a:buffer)
    endfunction


However, it seems to me that from user's perspective it would be better to add separate `rebar3_fmt` fixer.